src/styles/intent.styl

```diff
@@ -0,0 +1,52 @@
+@require 'settings/palette.styl'
+.app-wrapper
+    width 100%
+.app-content
+    height 100vh
+
+.intent-layout
+    height 100%
+    display flex
+    align-items stretch
+    flex-direction column
+
+
+.intent-footer
+    height 4.5rem
+    background-color paleGrey
+    box-shadow inset 0 -1px 0 0 silver
```

Comment:
Since we have 2 identical shadows, we should use a variable for this. I also believe our shadows are standardized and there are predefined ones in cozy-ui.

Reply:
I did not know that, thanks for the tips. Nevertheless I checked [cozy-ui docs](http://cozy.github.io/cozy-ui/styleguide) and I found nothing about shadows.
I cargo-cult this line from https://github.com/cozy/cozy-collect/blob/master/src/styles/services.styl#L51

Reply:
Looks like I'm wrong. i'm pretty sure there *used to be* box shadow variables, but I can't find them anymore either..?

Reply:
Not to my knowledge… maybe in the first experimental shot of Cozy-UI. It rings a bell somehow…